SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -93,13 +95,35 @@ private extension SponsorsView {
                 }
             }.foregroundColor(.clear)
     }
+    
+    func jobtile(for sponsor: Sponsor) -> some View {
+        ForEach(sponsor.jobs) { job in
+            CommonTileButton(primaryText: job.title,
+                             secondaryText: "💼",
```

Comment:
nit: I'm not sure a static emoji is enough to identify this as a role. I think this could be an interesting design element if it was personalised for the role (👨🏼‍💻, 🎨, 💼 - for developer, designer, and management respectively).